libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/SPManager.kt

```diff
@@ -126,7 +126,7 @@ internal class SPManager(
                 SalesforceSDKManager.getInstance().userAccountManager.switchToUser(user)
             }
 
-            override fun getMainActivityClass(): Class<out Activity>? {
+            override fun getMainActivityClass(): Class<out Activity> {
```

Comment:
Everything in this source is inspector cleanup for existing trivial issues and those introduced by the Kotlin conversion.